bigchaindb/common/schema/transaction.yaml

```diff
@@ -117,6 +117,25 @@ definitions:
         anyOf:
         - type: object
           additionalProperties: true
+          patternProperties:
+            language:
```

Comment:
Will this be applied to any `language` key within any nested document?
Also should we support two and three letter codes https://docs.mongodb.com/manual/reference/text-search-languages/?

Reply:
No. I think we can only handle this at the API level in #1806 . Also, I think it would be sufficient to just have the full names @ttmc What do you think?

Reply:
I'd prefer to allow both the long and short (two-letter) names for the languages, just like MongoDB.

Reply:
Sure, then we will keep the list as it is in #1806

Reply:
Maybe I am missing something but I don't understand the reason for this change.
This will not handle the case where the `language` is in a nested document and https://github.com/bigchaindb/bigchaindb/pull/1806 already covers all cases.

Reply:
@r-marques sorry for confusion, this pull request is not required anymore. We can close it.